JapaneseFilter/sections/specific.txt

```diff
@@ -35,6 +35,9 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+5chan.jp##div[class^="advertise_"] ([NSFW] https://c.5chan.jp/eu5FRL9Yr8)
+5chan.jp##.toppageadvheader
```

Comment:
You can restore the orignal rule as you see in the conversation.

Reply:
You can remove this and replace `5chan.jp##.toppageadvheader` in JPF with your original rule `5chan.jp##.socialandbanner`.

Reply:
Is this right?
https://github.com/AdguardTeam/AdguardFilters/pull/110500/commits/0bcbd0de56ae35399abf6435878fdc1857038034